bin/package

```diff
@@ -80,27 +80,14 @@ if [[ $CI != true && "$OSTYPE" == "darwin"* ]]; then
     yarn lint
 fi
 
-./node_modules/.bin/patch-package
+yarn patch-package
 
-(
-  cd packages/loot-design;
-  ../../node_modules/.bin/patch-package
-)
+yarn workspace mobile patch-package
 
-(
-  cd packages/mobile;
-  ../../node_modules/.bin/patch-package
-)
+yarn workspace loot-core build:node
 
-(
-  cd packages/loot-core;
-  NODE_ENV=production yarn build:node
-)
+yarn workspace @actual-app/web build
 
-(
-  cd packages/desktop-client;
-  yarn build
-)
```

Comment:
Why'd you delete this?

Reply:
These 4 lines are equivalent to running `yarn workspace @actual-app/web build` in the monorepo root. I felt that this script was more readable using this format rather than cding into each package.